### PR TITLE
refactor(play): use elm framework

### PR DESCRIPTION
**Pull request by youen**

### CODE REVIEW

**Comment by youen** [comment]
poke @adrienaury

**Comment by adrienaury** [comment]
Question about this warning in build logs

> warning " > elm-tailwind-modules@0.4.0" has incorrect peer dependency "tailwindcss@^2.0.2".

Is it possible to fix it ? What happen if it is not fixed ?

**Comment by adrienaury** [comment]
The default example was changed to 

```yaml
version: "1"
masking:
  - selector :
      jsonpath : "field_name"
    masks:
      - add: ""
```

from 

```yaml
version: "1"
masking:
  - selector:
      jsonpath: "name"
    mask:
      randomChoiceInUri: "pimo://nameFR"
```

Why ?

File: web/play/src/Tailwind/Breakpoints.elm [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module Tailwind.Breakpoints exposing (..)
```
Comment (adrienaury) [inline web/play/src/Tailwind/Breakpoints.elm:1]: Why this file and the other one in the Tailwind directory are commited ? They seems generated.

Comment (youen) [reply web/play/src/Tailwind/Breakpoints.elm]: i remove all generated files and add a command to recreate it

**Comment by youen** [comment]
> Question about this warning in build logs
> 
> > warning " > elm-tailwind-modules@0.4.0" has incorrect peer dependency "tailwindcss@^2.0.2".
> 
> Is it possible to fix it ? What happen if it is not fixed ?

Peer dependency doesn't accept tailwindcss > 3.0.0 but it should be supported. I create a new issue 
https://github.com/matheus23/elm-tailwind-modules/issues/14

**Comment by adrienaury** [comment]
@youen  The Zoom function on mermaid view is not working anymore

**Review by adrienaury — APPROVED** [review]

**Review by adrienaury — APPROVED** [review]

**Pull request merged**